package.json

```diff
@@ -28,17 +28,20 @@
     "@apollo/client": "3.6.9",
     "@emotion/react": "11.10.4",
     "@fullstacksjs/toolbox": "2.20.0",
-    "@mantine/core": "5.4.1",
-    "@mantine/hooks": "5.4.1",
```

Comment:
the build issue is with this line, the version should be as dev is 5.4.1

Reply:
Tnx a lot